packaging/CPackDebInstall.cmake

```diff
@@ -0,0 +1,79 @@
+# This file is executed during cpack time.
+# The command is
+# cpack -G DEB
+
+find_program( CPACK_DEBIAN_DEBHELPER dh_prep )
+if( NOT CPACK_DEBIAN_DEBHELPER )
+    message( FATAL_ERROR "debhelper not found, required for cpack -G DEB" )
+endif()
+
+find_program( CPACK_DEBIAN_MARKDOWN markdown )
+if( NOT CPACK_DEBIAN_MARKDOWN )
+   message( FATAL_ERROR "markdown not found, required for cpack -G External -D DEB_UPLOAD_PPA=true" )
+endif()
+
+find_program( CPACK_DEBIAN_DOCBOOK_TO_MAN docbook-to-man )
+if( NOT CPACK_DEBIAN_DOCBOOK_TO_MAN )
+    message( FATAL_ERROR "docbook-to-man not found, required for cpack -G DEB" )
+endif()
+
+if( NOT CPACK_DEBIAN_FAKEROOT )
+   message( FATAL_ERROR "fakeroot not found, required for cpack -G DEB" )
+endif()
+
+message( NOTICE "Creating debian folder" )
+file(COPY ${CPACK_DEBIAN_SOURCE_DIR}/packaging/debian
+    DESTINATION ${CPACK_TOPLEVEL_DIRECTORY}/${CPACK_PACKAGE_FILE_NAME})
+configure_file(${CPACK_TOPLEVEL_DIRECTORY}/${CPACK_PACKAGE_FILE_NAME}/debian/control.in
+               ${CPACK_TOPLEVEL_DIRECTORY}/${CPACK_PACKAGE_FILE_NAME}/debian/control
+               @ONLY)
+file(REMOVE ${CPACK_TOPLEVEL_DIRECTORY}/${CPACK_PACKAGE_FILE_NAME}/debian/control.in)
+
+file(COPY ${CPACK_DEBIAN_SOURCE_DIR}/res/linux/mixxx-usb-uaccess.rules
+    DESTINATION ${CPACK_TOPLEVEL_DIRECTORY}/${CPACK_PACKAGE_FILE_NAME}/debian)
+file(RENAME
+    ${CPACK_TOPLEVEL_DIRECTORY}/${CPACK_PACKAGE_FILE_NAME}/debian/mixxx-usb-uaccess.rules
+    ${CPACK_TOPLEVEL_DIRECTORY}/${CPACK_PACKAGE_FILE_NAME}/debian/mixxx.mixxx-usb-uaccess.udev)
```

Comment:
Can these be combined to one `file(COPY)` command?

Reply:
No, a file can only copied to a directory.